pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -134,6 +134,64 @@ func TestNewServer(t *testing.T) {
 	}
 }
 
+func TestPprof(t *testing.T) {
```

Comment:
This test will take 30s to run because of the profiling which is too long for a unit test. can we just check one of the fast ones like cmdline maybe?

Reply:
Of course.

Reply:
BTW, `/release-note-none` or `/release-notes-none` not work for me :(

Reply:
@daixiang0 you do not need to add release-notes-none label, please reference here: https://github.com/istio/istio/tree/master/releasenotes to add a release note, thanks!

Reply:
Done.